### ISSUE TITLE
Error: Cannot find module '@nestjs/graphql'

### ISSUE THREAD

**Issue by ErlanBelekov** [nestjs/graphql#2022]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current behavior

We have a NestJS GraphQL server as a package in a Yarn Workspaces monorepo.

Whenever I run it in dev or prod mode, it doesn't start the NestJS server.

```
Error: Cannot find module '@nestjs/graphql'
Require stack:
- /Users/imac/work/hub/node_modules/@nestjs/apollo/dist/drivers/apollo-federation.driver.js
- /Users/imac/work/hub/node_modules/@nestjs/apollo/dist/drivers/index.js
- /Users/imac/work/hub/node_modules/@nestjs/apollo/dist/index.js
- /Users/imac/work/hub/packages/server/dist/src/app.module.js
- /Users/imac/work/hub/packages/server/dist/src/main.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:902:15)
    at Function.Module._load (internal/modules/cjs/loader.js:746:27)
    at Module.require (internal/modules/cjs/loader.js:974:19)
    at require (internal/modules/cjs/helpers.js:93:18)
    at Object.<anonymous> (/Users/imac/work/hub/node_modules/@nestjs/apollo/dist/drivers/apollo-federation.driver.js:8:19)
    at Module._compile (internal/modules/cjs/loader.js:1085:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
    at Module.load (internal/modules/cjs/loader.js:950:32)
    at Function.Module._load (internal/modules/cjs/loader.js:790:12)
    at Module.require (internal/modules/cjs/loader.js:974:19)
```

### Minimum reproduction code

1. Create a new yarn workspaces project
2. Create a NestJS server as a package in `packages/server`
3. Add graphQL packages as described here, and init GraphQLModule like so, with a code-first approach:
```
// code-first GQL module
GraphQLModule.forRoot<ApolloDriverConfig>({
  driver: ApolloDriver,
  autoSchemaFile: 'src/transport/graphql/schema.graphql',
  sortSchema: true,
  context: createContext,
  cors: {
    origin: process.env.CORS_ORIGIN,
    credentials: true,
  },
}),
```

### Steps to reproduce

1. `yarn`
2. `cd packages/server`
3. `yarn start:dev`

### Expected behavior

should start the Nest server.

### Package version

10.0.2

### Graphql version

`graphql`: 16.3.0
`apollo-server-express`: 3.6.3
`apollo-server-fastify`: 

### NestJS version

8.0.0

### Node.js version

v14.18.2

### In which operating systems have you tested?

- [X] macOS
- [ ] Windows
- [ ] Linux

**Comment by jmcdo29** [comment]
Please provide a [minimum reproduction repository](https://minimum-reproduction.wtf)

**Comment by barnc** [comment]
Same issue here

**Comment by kamilmysliwiec** [comment]
Fixed in 10.0.3

**Comment by quyetinforma** [comment]
Heya, I got the same issue even with latest release 10.0.11, here is my dependencies. How to fix this?
`"dependencies": {
    "@apollo/gateway": "0.46.0",
    "@nestjs/apollo": "9.2.4",
    "@nestjs/common": "8.2.3",
    "@nestjs/core": "8.2.3",
    "@nestjs/graphql": "10.0.11",
    "@nestjs/platform-express": "8.2.3",
    "apollo-server-express": "3.6.2",
    "graphql": "15.7.2",
    "graphql-tools": "8.0.0",
    "reflect-metadata": "0.1.13",
    "rimraf": "3.0.2",
    "rxjs": "7.4.0",
    "ts-morph": "12.2.0"
  },`
